admin/rsyslog/files/rsyslog.init

```diff
@@ -5,8 +5,100 @@ START=20
 
 USE_PROCD=1
 
+UCI_CONF="rsyslog"
+CONFIG_FILE="/etc/rsyslog.conf"
+
+modules=""
+selectors=""
+forwarders=""
+
+handle_selector() {
+    local config="$1"
+    local src
+    local dst
+
+    config_get src ${config} source
+    config_get dst ${config} destination
+    if [ ${src} != "" ] && [ "$dst" != "" ]; then
+        selectors="${selectors}\n${src}\t${dst}\n"
+    fi
+}
+
+handle_forwarder() {
+    local config="$1"
+    local src
+    local target
+    local protocol
+    local port
+    local rfc
+    local opts
+
+    config_get src ${config} source
+    config_get target ${config} target
+    config_get protocol ${config} protocol "udp"
+    config_get port ${config} port "514"
+    config_get rfc ${config} rfc "3164"
+
+    if [ "$rfc" == "5424" ]; then
```

Comment:
== is a bashism. Please run through shellcheck.

Reply:
I've already done that, but disabled the check because it works with `sh`.
Still, I've added the change as request: https://github.com/openwrt/packages/pull/18714/commits/3db02aa66210cf623bff61f845774594ab8b7a78

Also fixed the commit formalities: https://github.com/openwrt/packages/runs/7208594812?check_suite_focus=true#step:4:53